src/profile/components/AgeMessage.jsx

```diff
@@ -1,12 +1,9 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { StatusAlert } from '@edx/paragon';
 import { FormattedMessage } from 'react-intl';
 
-import { configuration } from '../../config/environment';
-
-const { ACCOUNT_SETTINGS_URL } = configuration;
```

Comment:
Example of the tight coupling we had before.  AgeMessage should definitely not know there's some random file outside its module called environment.js where it can get a URL it needs.